pgamit/pyETM.py

```diff
@@ -2340,8 +2401,6 @@ def enable_picking(self, event):
     def plot_hist(self, pngfile=None, fileio=None):
 
         import matplotlib.pyplot as plt
```

Comment:
we'll probably want to move these imports to the top of the module instead of having them executed at function runtime...